manageiq-providers-vmware.gemspec

```diff
@@ -14,7 +14,7 @@ Gem::Specification.new do |s|
   s.files = Dir["{app,config,lib}/**/*"]
 
   s.add_dependency("fog-vcloud-director", ["~> 0.3.0"])
-  s.add_dependency "vmware_web_service",      "~>0.4.3"
+  s.add_dependency "vmware_web_service",      "~>0.4.10"
```

Comment:
Should we just do `"~>0.4"`?  I wasn't sure

Reply:
Also, since we are loading `ffi-vix_disk_lib` directly...should we add it as a dependency directly here?

Reply:
Yeah I think ~>0.4 is better and :+1: on adding ffi-vix_disk_lib